### PR TITLE
Temporal Augmentation experiments + normalization updates

### PR DESCRIPTION
**Pull request by ivanzvonkov**
Experiments outlined here: https://wandb.ai/nasa-harvest/temporal-augmentation/reports/Temporal-Augmentation-Issues--VmlldzoxNjQ2NDE5

### CODE REVIEW

File: src/utils/data_transform.py [hunk 1]
```diff
@@ -124,6 +133,10 @@ class PlanetTransform(EOTransformer):
     THIS CLASS INHERITS EOTRANSFORMER FOR DATA AUGMENTATION IN THE PLANET DATA
     """
 
+    # South Africa values
+    per_band_mean = np.array([580.4186, 852.98376, 1136.9423, 2761.0286])
+    per_band_std = np.array([179.95744, 209.66647, 384.34073, 476.9446])
```
Comment (ivanzvonkov) [inline src/utils/data_transform.py:138]: @joos2010kj we'll need similar values for Germany here

File: visualize_ta.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import torch
+import wandb
+
+from typing import List
+
+from src.utils.data_transform import PlanetTransform
+
+
+def get_ndvi(image_stack):
+    PlanetTransform.per_band_mean
```
Comment (joos2010kj) [inline visualize_ta.py:12]: What is it doing standalone?

**Review by joos2010kj — APPROVED** [review]
ngl it was a big temp aug commit, and code-wise it looked good, but I will have to run it to make sure it's working fine too.

File: visualize_ta.py (hunk 2 above)
Comment (ivanzvonkov) [reply visualize_ta.py:12]: This is a helper function for making these plots
image
